### DISCUSSION TITLE
Compatibility with MotoGP

### DISCUSSION THREAD

**Discussion by Jesper095** [f1multiviewer/issue-tracker#470] [Feature requests]
Adding compatibility with MotoGP would be cool

**Comment by amstoneberger** [comment]
@f1multiviewer reviving this thread

**Comment by LapsTimeOFF** [comment]
Hey, going to quote MV from a few days ago on our Discord:
<img width="566" height="306" alt="image" src="https://github.com/user-attachments/assets/3cd224d4-4353-430e-a48f-db005e960e45" />
